test/lib/creation.go

```diff
@@ -428,6 +429,8 @@ func (c *Client) CreatePodOrFail(pod *corev1.Pod, options ...func(*corev1.Pod, *
 		}
 	}
 
+	pkgsecurity.AllowRestrictedPodSecurityStandard(context.Background(), c.Kube, pod)
```

Comment:
This looks good.

Any reason this PR is "WIP"  ? 

I think it would be nice to be backporting this too?